NextTime.yaml

```diff
@@ -18,4 +18,5 @@ suggestions:
   - Add git Kraken guide if not done so already
   - Markup
   - CMake
+  - WSL guide
```

Comment:
Was there anything in particular or just WSL in general? I can try to find a good guide to add to the Learning Resources page on NUbook. A lot of us use Linux since it's much easier than dealing with WSL, but there might be some people who can help with WSL.

Reply:
> Ah, sorry, should have been clearer - the title won't have the words 'This PR will", just that it would finish the sentence. It's more of a trick to help you write a good PR title. So this title would be "Update foods.yaml with macadamias & update NextTime.yaml with added suggestion of WSL guide".

Ah, I see now. No worries, got you.

Reply:
> 

I guess particularly would be using it on VS code with Github, but I'm also understanding now that there are many other platforms that probably are simpler to use.